### PR TITLE
[Pavlo Turchyn]tasks:81-90

### PR DESCRIPTION
**Pull request by FabianMoretti**

### CODE REVIEW

File: tasks/84_added-characters/added-characters.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* eslint "require-jsdoc": 0 */
+export const addedChar = (string1, string2) => {
+  string1 = string1.split('').sort().join('');
+  string2 = string2.split('').sort().join('');
+  const str1Length = string1.length;
+  const str2Length = string2.length;
+  for (let i = 0; i < str1Length; i++) {
+    if (string1[i] !== string2[i]) {
```
Comment (AlyonaPianykh) [inline tasks/84_added-characters/added-characters.js]: Don't put value of `array.length` in a separate variable, you can use it in the for loop condition`i < string1.length;`

File: tasks/85_bet-validation/bet-validation.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+/* eslint-disable-next-line */
+export const validate_bet = ([count, max], str) => {
+  let str1 = str.split(/,| /);
+  str1 = str1.filter((item) => item !== '');
+  for (let i = 0; i < str1.length; i++) {
+    if (isNaN(+str1[i])) {
+      return 'None';
+    }
+  }
+  str1 = str1.map((item) => +item);
```
Comment (AlyonaPianykh) [inline tasks/85_bet-validation/bet-validation.js]: You can use just one `.map` loop right after `.filter`, `for` loop is unnecessary. 

Please refactor this function to make it simpler.

File: tasks/87_validate-us-telephone-number/validate-us-telephone-number.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    type3: /^[1][ ][0-9]{3}[-][0-9]{3}[-][0-9]{4}/,
+    type4: /^[1][ ][0-9]{3}[ ][0-9]{3}[ ][0-9]{4}/,
+    type5: /^[1][ ][(][0-9]{3}[)][ ][0-9]{3}[-][0-9]{4}/,
+    type6: /^[(][0-9]{3}[)][0-9]{3}[-][0-9]{4}/,
+    type7: /^[(][0-9]{3}[)][ ][0-9]{3}[-][0-9]{4}/,
+    type8: /^[1][(][0-9]{3}[)][0-9]{3}[-][0-9]{4}/,
+  };
+  /* eslint-disable-next-line */
+  for (let value in types) {
+    if (types['type2'].test(str) && str.length !== 10) return false;
```
Comment (AlyonaPianykh) [inline tasks/87_validate-us-telephone-number/validate-us-telephone-number.js]: Why do you need this condition?

File: tasks/88_record-collection/record-collection.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    },
+    '1245': {
+      'artist': 'Robert Palmer',
+      'tracks': [],
+    },
+    '5439': {
+      'album': 'ABBA Gold',
+    },
+  };
+  let collectionCopy = JSON.parse(JSON.stringify(collection));
```
Comment (AlyonaPianykh) [inline tasks/88_record-collection/record-collection.js]: This line doesn't make any sense. You can create copy of object by using for example spread operator, you don't need to use JSON.parse and JSON.stringify for this purpose.

File: tasks/89_symmertic-difference/symmertic-difference.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+/* eslint "require-jsdoc": 0 */
+export function sym() {
+  let arr = [];
+  let simddif = [];
+  const argsLength = arguments.length;
+  for (let i = 0; i < argsLength; i++) {
+    /* eslint-disable-next-line */
+    arr.push(unique(arguments[i]));
+  }
```
Comment (AlyonaPianykh) [inline tasks/89_symmertic-difference/symmertic-difference.js]: Arguments is pseudo array, it's a good practice to create a normal array from arguments before working with them. 
And once again, please don't create separate variable to store array length in it.

File: tasks/88_record-collection/record-collection.js (hunk 4 above)
Comment (AlyonaPianykh) [reply tasks/88_record-collection/record-collection.js]: Same code  to your colleague Sofia's

**Comment by FabianMoretti** [comment]
Так, це її код

Пн, 29 жовт. 2018 22:31 користувач AlyonaPianykh <notifications@github.com>
пише:

**Pull request merged**